admin-dev/themes/new-theme/scss/components/layout/_header_toolbar.scss

```diff
@@ -82,8 +80,7 @@
     position: relative;
 
     .notification-container {
-      @include notification_container(1.5rem)
-      position: absolute;
+      @include notification_container(1.5rem) position: absolute;
```

Comment:
Why remove the EOL after the include ?

Reply:
God, that should be a prettier misconfiguration, I've to fix that :)